packages/language-server/test/documentation-spec.ts

```diff
@@ -0,0 +1,83 @@
+import { expect } from "chai";
+import {
+  buildUI5Enum,
+  generateModel,
+} from "@ui5-language-assistant/test-utils";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+import { getNodeDocumentation } from "../src/documentation";
+
+describe("The @ui5-language-assistant/language-server <getNodeDocumentation> function", () => {
+  let ui5SemanticModel: UI5SemanticModel;
+  before(async function () {
+    //TODO: use 1.71.x
+    ui5SemanticModel = await generateModel({ version: "1.74.0" });
+  });
+
+  context("deprecatedInfo", () => {
+    it("will get documentation with deprecatedInfo", () => {
+      const ui5Enum = buildUI5Enum({
+        name: "dummy-node",
+        deprecatedInfo: { isDeprecated: true, since: "2.2.2", text: "" },
+      });
+      const result = getNodeDocumentation(ui5Enum, ui5SemanticModel);
+      expect(result.value).to.include("Deprecated since version 2.2.2.");
+    });
+
+    it("will get documentation with deprecatedInfo - without since", () => {
+      const ui5Enum = buildUI5Enum({
+        name: "dummy-node",
+        deprecatedInfo: {
+          isDeprecated: true,
+          since: undefined,
+          text: undefined,
+        },
+      });
+      const result = getNodeDocumentation(ui5Enum, ui5SemanticModel);
+      expect(result.value).to.include("Deprecated.");
+      expect(result.value).to.not.include("since");
+    });
+  });
+
+  context("experimentalInfo", () => {
+    it("will get documentation with experimentalInfo", () => {
+      const ui5Enum = buildUI5Enum({
+        name: "dummy-node",
+        experimentalInfo: {
+          isExperimental: true,
+          since: "2.2.2",
+          text: undefined,
+        },
+      });
+      const result = getNodeDocumentation(ui5Enum, ui5SemanticModel);
+      expect(result.value).to.include("Experimental since version 2.2.2.");
+    });
+
+    it("will get documentation with experimentalInfo - without since", () => {
+      const ui5Enum = buildUI5Enum({
+        name: "dummy-node",
+        experimentalInfo: {
+          isExperimental: true,
+          since: undefined,
+          text: undefined,
+        },
+      });
+      const result = getNodeDocumentation(ui5Enum, ui5SemanticModel);
+      expect(result.value).to.include("Experimental.");
+      expect(result.value).to.not.include("since");
+    });
+
+    it("will get documentation with experimentalInfo - without since", () => {
```

Comment:
I think this test appears twice
`it("will get documentation with experimentalInfo - without since", () => {`